resources/Microsoft.Windows.Settings/source/en-US/Microsoft.Windows.Settings.strings.psd1

```diff
@@ -0,0 +1,17 @@
+<#
+    .SYNOPSIS
+        The localized resource strings in English (en-US) for the
+        resource Microsoft.Windows.Settings module. This file should only contain
+        localized strings for private and public functions.
+#>
+
+ConvertFrom-StringData @'
+    ## Get-RegistryStatus
+    GetRegistryStatusRegistryManualManipulation_ErrorMessage = "The registry path '{0}' is not in the expected state. Entries have been manually modified. Cannot determine the current state."
```

Comment:
I couldn't find usage of some of those names. Do we need to remove them from this file?
- GetRegistryStatusRegistryManualManipulation_ErrorMessage
- GetRegistryStatus_SearchMessage
- GetRegistryStatus_FoundMessage